components/03-organisms/layout/two-column/_two-column.scss

```diff
@@ -0,0 +1,57 @@
+@use '../../../00-tokens/tokens';
+
+$break-layout-two-column: tokens.$break-2xl;
+$break-layout-two-column-max: $break-layout-two-column - 0.05;
+
+.yds-two-column {
+  @include tokens.spacing-page-section;
+}
+
+.yds-two-column__inner {
+  @media (min-width: $break-layout-two-column) {
```

Comment:
Not a comment that's a blocker for merging but I think these styles could actually be merged into the generic `[class*="__inner"]` styles. For the most part you are adding these styles for individual components already. 
When I did this locally I did a quick run through of the components and there were some minor adjustments I needed to make on several components (Example: page-title need a `flex-flow: column nowrap;` ) but otherwise it did look pretty good. The only component I saw that would need some work was the `Pop Out Image` because it's trying to utilize floats.
Just thought I would pass this thought along on a way to possibly make things more efficient.